js/views/chatHeadsVw.js

```diff
@@ -88,7 +103,12 @@ module.exports = baseVw.extend({
       );
     });
 
-    this.nextChatToShow = this.nextChatToShow >= totalChats ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+
+    this.nextChatToShow = this.nextChatToShow >= this.collection.length ? this.nextChatToShow : this.nextChatToShow + this.showPerScroll;
+  },
+
+  remove: function(){
```

Comment:
Don't forgot to call the remove function in the baseVv. See some other example of where remove is implemented in a view that extends from baseVw.